_docs_integrate/version-5.md

```diff
@@ -99,6 +99,60 @@ In addition, in contrast to the RequestItems, a RequestItemGroup did not have to
 The `mustBeAccepted` property of the RequestItems will be kept.
 {: .notice--info}
 
+## New mandatory wrappers for non-standard content of RelationshipTemplates, Relationships and Messages
+
+New wrapping types for sending arbitrary content that does not fit to standard enmeshed-content types have been introduced, the [ArbitraryMessageContent]({% link _docs_integrate/data-model-overview.md %}#arbitrarymessagecontent) for Messages with non-standard content, the [ArbitraryRelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#arbitraryrelationshiptemplatecontent) for RelationshipTemplates and the [ArbitraryRelationshipCreationContent]({% link _docs_integrate/data-model-overview.md %}#arbitraryrelationshipcreationcontent) for Relationships (the creationContent itself is [newly introduced here](#removal-of-relationshipchanges)).
+
+The `content` of a [Message]({% link _docs_integrate/data-model-overview.md %}#message) must either be a [Mail]({% link _docs_integrate/data-model-overview.md %}#mail), [Notification]({% link _docs_integrate/data-model-overview.md %}#notification), [Request]({% link _docs_integrate/data-model-overview.md %}#request), [ResponseWrapper]({% link _docs_integrate/data-model-overview.md %}#responsewrapper) or an [ArbitraryMessageContent]({% link _docs_integrate/data-model-overview.md %}#arbitrarymessagecontent).
+
+The `content` of a [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate) must either be a [RelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#relationshipcreationcontent) or an [ArbitraryRelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#arbitraryrelationshiptemplatecontent).
+
+The `creationContent` of a [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) must either be a [RelationshipCreationContent]({% link _docs_integrate/data-model-overview.md %}#relationshipcreationcontent) or an [ArbitraryRelationshipCreationContent]({% link _docs_integrate/data-model-overview.md %}#arbitraryrelationshipcreationcontent).
+
+Those new wrappers all have the type
+
+```jsonc
+{
+  "@type": "Arbitrary<...>Content",
+  "value": any,
+}
+```
+
+Thus, when sending a Message with non-standard content via POST `Messages`, instead of sending the parameter `content : <arbitrary-content>`, you send
+
+```jsonc
+content: {
+  "@type": "ArbitraryMessageContent",
+  "value": <arbitrary-content>
+}
+```
+
+Similarly, when creating a RelationshipTemplate with non-standard content via POST `RelationshipTemplates`, you send the parameter
```

Comment:
If we mention the routes instead of use cases, then please use the complete `POST /api/v2/RelationshipTemplates/Own`.